pkg/virt-controller/services/template.go

```diff
@@ -69,6 +69,8 @@ const NFD_CPU_FEATURE_PREFIX = "feature.node.kubernetes.io/cpu-feature-"
 
 const MULTUS_RESOURCE_NAME_ANNOTATION = "k8s.v1.cni.cncf.io/resourceName"
 
+const ISTIO_KUBEVIRT_ANNOTATION = "traffic.sidecar.istio.io/kubevirtInterfaces"
```

Comment:
can we get a comment in the code that gives us a hint about what this annotation does?

Reply:
Yes for sure I will add